### DISCUSSION TITLE
Hi! I'm Claas from Munich :)

### DISCUSSION THREAD

**Discussion by T4B4** [OpenMined/courses#27] [Introduce Yourself]
Hello Everyone! Quite excited to be here!

My name is Claas and I am currently writing my Masters thesis titled "Federated Learning for Prediction Tasks in Autonomous Systems", mostly focusing on applications in the mobility sector. I am currently looking for academic collaboration partners abroad. Besides this I am looking into startup opportunities (mostly other than autonomous driving) as I intend to stay in this field after graduation. 

I am also part of a student driven think tank called ThinkTech, in which I founded the Data Economy & Attention Economy topic group with the aim to understand how data and derivative data structures are manifesting as economic assets today and in the future. 

If I can help you with anything please let me know and don't hesitate to reach out!

https://www.linkedin.com/in/bruess/

<a href="https://imgflip.com/i/4sbzoo"><img src="https://i.imgflip.com/4sbzoo.jpg" title="made at imgflip.com"/></a><div>

**Comment by jS5t3r** [comment]
Hi Class, 

I am Peter. Living 1hour northly from Munich :) Nice to meet you!

**Reply by T4B4** [reply]
Awesome! Great to meet you. Are you already in the openmined slack?

**Reply by jS5t3r** [reply]
Yes, my name there is j53t3r

**Comment by LourensWalters** [comment]
Hi Claas, great to meet you. Love the photos!

**Comment by iamtrask** [comment]
Hi Claas! Welcome!
